rules/sinks/third_parties/sdk/quora/javascript.yaml

```diff
@@ -0,0 +1,14 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+
+  - id: ThirdParties.SDK.Pixel.Quora
+    name: Quora
+    domains:
+      - "quora.com"
+    filterProperty: "code"
+    patterns:
+      - "(?i)(.*[.])?qp[(].*"
```

Comment:
Specially two letter matches like one and `fs` in one of the other regexes, I feel that these are super wide and could end up matching a lot of other things, if possible, we should be more specific. For e.g. this regex will match `"anythingqp("` as a string, I know 'qp' don't occur together often but still.

Reply:
it would either match "anything.qp(" or just "qp("

Reply:
The `[.]` makes it any character, if we really want just `.`, then we should escape it.

Reply:
yes right, this is more wider and could result in many false positive. @HiteshMah-Jan can suggestion?

Reply:
we are finally replacing `[.]` with `//.`

Reply:
No. A dot in a box will only match a dot. anything inside `[]` are not treated as special chars.

Reply:
I think both `[.]` or `\\.` will give same result. it good to have `[.]` as this will lead to less regex related issue.